fuse/cross_agency.py

```diff
@@ -104,6 +110,19 @@ def cross_match_officers_between_agencies(personnel, events, constraints):
     per = discard_rows(
         per, per.agency != "", "officers not linked to any event", reset_index=True
     )
+
+    per = pd.merge(
+        per,
+        post,
+        on=["uid", "first_name", "last_name", "agency"],
+        how="outer",
+    )
```

Comment:
Do we need to include `first_name` and `last_name` when `uid` is already present? `post` probably contains multiple rows with the same `uid`. I wonder what effects would that have on the output. It's safer to deduplicate `post` on `uid` first.

Reply:
Got it. Thanks.